### PR TITLE
Explain circles/contacts for invite

### PR DESCRIPTION
**Pull request by decentral1se**
Attempting to explain this automagic contact/circle logic from the first steps.

Perhaps missing a glossary page for explaining what a circle is?

### CODE REVIEW

**Comment by mwild1** [comment]
I have a half-written doc explaining circles in detail, for the server operators guide.

Right now, although this doc is overall good, I'm on the fence about exposing the "circle" term to users. Currently it is nowhere in the Snikket UI, unless you're the server admin. Hmm...

**Comment by decentral1se** [comment]
Ah, I see. The reason I ended up adding this because I was trying to explain the server setup to some curious users. They wanted to know who else was here on the server and if it was OK to make additional group chats. Also, an experienced XMPP user wanted to know why a bunch of people were already added as friends in their roster. I know some/most users wont care if things Just Work :tm: but when these seemingly easy to answer questions end up with having to refer to a "circle", I feel like it might be nice to have something upstream here in the client docs to help. Unsure myself anyway...

File: content/getting-started/accept-invitation.md [hunk 1]
```diff
@@ -39,3 +39,14 @@ invitation page again, but this time tap the 'Open the app' button on that
 page to launch the app and begin the account setup.
 
 As part of the account setup process, you will choose a username and password.
+
+## Step 4: Understanding circles and contacts
+
+Once you've created your account and opened the Snikket app, you'll see that
+you've automatically been invited to a group chat.
```
Comment (mwild1) [inline content/getting-started/accept-invitation.md]:
```suggestion
you've automatically been added to a group chat.
```

Snikket apps auto-accept invites, but the circle group functionality doesn't actually use invites anyway (the group is added directly to the user's bookmarks).

**Review by mwild1 — CHANGES_REQUESTED** [review]

File: content/getting-started/accept-invitation.md (hunk 1 above)
Comment (decentral1se) [reply content/getting-started/accept-invitation.md]: Ah yep, fixed up @mwild1.

**Review by mwild1 — APPROVED** [review]

**Pull request merged**